guides/quickstart.md

```diff
@@ -4,426 +4,305 @@ title: Quickstart
+Now we have a pretty solid and easy to use calculator testing API.
 
-```js
-test('calls submit action with correct username and password', function(assert) {
-  assert.expect(2);
+We don't rely on any CSS selectors directly in our tests. We've also absorbed DOM layout
+complexity and exposed only functional properties of the component to the test.
 
-  function submit(username, password) {
-    assert.equal(username, 'admin');
-    assert.equal(password, 'secret');
-  }
+All this makes a test suite easier to maintain and fun to implement!
 
-  this.set('submit', submit);
+## Pages
 
-  this.render(hbs`{{login-form
-    submit=(action submit)
-  }}`);
+A web page is usually associated with a certain resource URL which is represented
+by some components hierarchy. Having an URL attribute makes testing of application
+pages a bit special.
+
+Let's take a look on how the generated page-object page does look like:
 
-  page.loginSuccessfully();
-});
 ```
+$ ember generate page-object my-page
 
-And that's it! Our integration and acceptance tests are cleaner, more maintainable and easier to read.
+installing
+  create tests/pages/my-page.js
+```
 
-## Appendix
+The generator created a file inside the directory `/tests/pages`.
+It allows to easier distinguish pages between components, which are
+located under `/tests/pages/components/`.
 
-A helpful tip is to separate the exports in component page objects. This will allow you to compose larger page objects using the same definitions. For example say we have an integration test of a `my-fanfare` component:
+A generated page output looks like the following:
 
 ```js
-import {
-  create,
-  clickable,
-  isVisible
-} from 'ember-cli-page-object';
+// my-app/tests/pages/my-page.js
+import { create, visitable } from 'ember-cli-page-object';
 
-export const MyFanfare = {
-  scope: '.ui-my-fanfare',
-  playFanfare: clickable('button'),
-  isCelebrating: isVisible('.fireworks')
-};
-
-export default create(MyFanfare);
+export default create({
+  visit: visitable('/')
+});
 ```
 
-This separation gives us two `import`-able signatures. In the case of the component's integration test importing the `default` will work as expected:
+The page is provided with a [`visitable`](./api/visitable) attribute to open
+associated web page.
 
-```js
-import { moduleForComponent, test } from 'ember-qunit';
-import hbs from 'htmlbars-inline-precompile';
-import page from 'my-app/tests/pages/components/my-fanfare';
-
-moduleForComponent('my-fanfare', 'Integration | Components | my fanfare', {
-  integration: true,
-  beforeEach() {
-    page.setContext(this);
-  },
-  afterEach() {
-    page.removeContext();
-  }
-});
+You also might have noticed that rather than exporting a plain definition we
+export a page object instance which can be used in your tests right after the page
+is imported.
```

Comment:
Yes, this place probably needs an additional explanation. I think a reason for exporting a ready to use page object instances is that we don't need them to be extended.

> for performance?

Not sure what do you mean under performance..

Reply:
I presumed the reason for advocating exporting page objects rather than definitions was for performance, since creating page object singletons once will be more performant than creating page objects per individual test case.

Reply:
I believe, creating page object instances has very low impact on the test-suite performance. I don't believe we can slowdown a test-suite even if we create page object instances per each test run 😆 

I think that the reason for advocating for exporting of page objects is just a fact that a specific app page has a well known shape(set of components, URL, ...) and it rarely needs to be extended.